geode-docs/developing/query_additional/operators.html.md.erb

```diff
@@ -27,25 +27,31 @@ Comparison operators compare two values and return the results, either TRUE or F
 
 The following are supported comparison operators:
 
-|          |                          |
+| Operator | Meaning                  |
 |----------|--------------------------|
-| =        | equal to                 |
-| &lt;&gt; | not equal to             |
-| !=       | not equal to             |
 | &lt;     | less than                |
 | &lt;=    | less than or equal to    |
 | &gt;     | greater than             |
 | &gt;=    | greater than or equal to |
+|          |                          |
+| =        | equal to                 |
+| !=       | not equal to             |
+| &lt;&gt; | not equal to             |
 
-The equal and not equal operators have lower precedence than the other comparison operators. They can be used with null. To perform equality or inequality comparisons with UNDEFINED, use the IS\_DEFINED and IS\_UNDEFINED preset query functions instead of these comparison operators.
+Regarding equality and inequality operators:
+
+- The equal and not equal operators have lower precedence than the other comparison operators.
+- The equal and not inequal operators can be used with null.
```

Comment:
Should this be "The equal and not-equal operators"?

Reply:
equality and inequality operators I think is the term we are looking for here..